docs/5_1_scheduled_execution_math.adoc

```diff
@@ -5,25 +5,12 @@
 In Scheduled Execution the importer has to ensure that model partitions are scheduled according to their activation time and priorities.
 This is realized in <<ClockActivationMode>> by calling <<fmi3ActivateModelPartition>> triggering their respective Clocks.
 
-After <<fmi3ActivateModelPartition>> has been called for a <<fixed-clock,`fixed`>>, <<tunable-clock,`tunable`>> or <<changing-aperiodic-clock,aperiodic>> Clock the FMU provides the information on when the Clock will tick again, i.e. when the corresponding model partition must be scheduled the next time.
-To retrieve this information the importer calls <<fmi3GetInterval>> on the Clock.
-
 The FMU may request to schedule another model partition even while currently a model partition is being executed.
 
-A new interval for a <<countdown-aperiodic-clock,`countdown`>> Clock can be provided at any time during the execution of any model partition.
-The FMU signals that a <<countdown-aperiodic-clock,`countdown`>> Clock is about to tick by calling <<fmi3ClockUpdateCallback>>.
-The importer calls <<fmi3GetInterval>> for all <<countdown-aperiodic-clock,`countdown`>> Clocks.
-If a new interval latexmath:[T_{\mathit{interval, i}}] is provided, i.e. return argument <<qualifiers>> = <<fmi3IntervalChanged>>, the importer initiates the scheduling of the associated model partitions regarding the delay of the returned interval.
-
 In case more than one Clock ticks at the same time instant, the scheduler needs a priority to define the activation sequence of the associated model partitions.
 This ordering is defined by the <<priority>> attributes of the Clock.
 For Scheduled Execution this attribute is mandatory for every Clock.
 
-_[A clock's <<priority>> information is required to support the scheduled execution of several model partitions._
```

Comment:
Why is this deleted? Or is it moved? Hard do figure out in such a huge PR ;)

Reply:
See Clocks section - there it is complete and correct - we wanted to remove the redundancy with slight differences.